timechart/tests/striptool_config_import/output/strip.stp.json

```diff
@@ -1,5 +1,5 @@
 {
-    "__version__": "1.1.0+24.g6ff00aa.dirty",
+    "__version__": "1.2.1+32.g48e0c87.dirty",
```

Comment:
These supposed to be in version control? If not best ignore them with `.gitignore`.